code/game/gamemodes/game_mode.dm

```diff
@@ -360,6 +361,11 @@
 					if(age_check(player.client)) //Must be older than the minimum age
 						candidates += player.mind				// Get a list of all the people who want to be the antagonist for this round
 
+	if(protected_species)

```

Comment:
uh, just check this when you build the candidate list, what the hell is this wasteful crap

Reply:
already fixed, I just need to push the changes.

Reply:
when are you going to do it